packages/terra-dev-site/src/webpack/plugin/applyDefaults.js

```diff
@@ -53,6 +53,7 @@ const applyDefaults = ({
   enableDebugLogging = false,
   additionalConfig = {},
   excludeChunks = [],
+  contentDirectory = 'terra-dev-site',
```

Comment:
I modified this to allow consumers to change the root content directory for their plug in via config. Previously you had to make a new 'plugin' to modify this. I used this to break out the terra-dev-site test site into it's own folder.